core/templates/dev/head/editor/EditorServices.js

```diff
@@ -1244,9 +1246,15 @@ oppia.factory('explorationGadgetsService', [
         return;
       }
       _gadgets[gadgetData.gadget_name] = gadgetData;
-      _panels[panelName].push(gadgetData.gadget_name);
+      _panels[gadgetData.gadget_panel].push(gadgetData.gadget_name);
       $rootScope.$broadcast('gadgetsChangedOrInitialized');
-      changeListService.addGadget(gadgetData, panelName);
+      // REFACTORING(anuzis): panelName formerly passed as 2nd arg below.
+      // Temporarily passing from shared encapsulating variable to see if it
+      // works before refactoring further downstream.
+      // UPDATE: Confirmed this works. @sll: What's your feedback on this
```

Comment:
This is fine. I agree changeListService.addGadget() only needs to take a single arg.